### DISCUSSION TITLE
Suggestion 002/100 : The Backup Option.

### DISCUSSION THREAD

**Discussion by nanohtpc** [Chia-Network/chia-blockchain#17712] [Ideas]
We need a backup option guyz....

or the ability to export / import values from the nodes between computers...

- the keys
- the contacts
- the settings
- the wallets
etc...

**Comment by BrandtH22** [comment]
Hey @nanohtpc , there is a database backup command but otherwise one should always have their mnemonics backed up which give access to the wallets and keys. 

As for the contact list that is current only part of the gui and a great feature request! If you wanted to update the title of this request to focus on the backing up of the contact list that would be a great enhancement!

As always with feature requests our engineers will review and work on them as time permits and when applicable so there are no guarantees features will be built or be built on any specific timeline.

**Comment by nanohtpc** [comment]
just quick reading.. im gonna answer later today....

Sent with [Proton Mail](https://proton.me/) secure email.
